src/dep/fmt/fmt.vcxproj

```diff
@@ -16,10 +16,22 @@
     </ClCompile>
   </ItemDefinitionGroup>
   <ItemGroup>
-    <ClCompile Include="$(SolutionDir)\oss\fmt\src\*.cc" />
+    <ClCompile Include="$(SolutionDir)oss\fmt\src\format.cc" />

```

Comment:
Should we add a note to `MAINTAINER_README.md` that updates will require checking this file now?